libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -898,6 +902,15 @@ DNNModel *ff_dnn_load_model_tf(const char *model_filename, DNNFunctionType func_
         goto err;
     }
 
+    tf_model->task_queue = ff_queue_create();
+    if (!tf_model->task_queue) {
+        goto err;
+    }
+
+#if !HAVE_PTHREAD_CANCEL
+    av_log(ctx, AV_LOG_WARNING, "pthreads not supported. Roll back to sync.\n");
```

Comment:
#if !HAVE_PTHREAD_CANCEL
if (ctx->options.async)
   av_log(...)
#endif

Reply:
The async is in the DnnContext which isn't being passed to the load_model function. I also found this line in `dnn_filter_common.c`:
```c
int ff_dnn_init(DnnContext *ctx, DNNFunctionType func_type, AVFilterContext *filter_ctx)
{
...
#if !HAVE_PTHREAD_CANCEL
    if (ctx->async) {
        ctx->async = 0;
        av_log(filter_ctx, AV_LOG_WARNING, "pthread is not supported, roll back to sync.\n");
    }
#endif
...
}
```
So there is no need for checking this for now :thinking:, but I think we'll need this line in the unification patches. Shall I remove this line for now?

Reply:
yes, as for now, we may remove it, thanks.

Reply:
Thanks, sent.